packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -131,6 +145,67 @@ export default class QuanticSort extends LightningElement {
     this.hasInitializationError = true;
   }
 
+  /**
+   * Sets the error when custom sort options have an invalid configuration.
+   */
+  setSortOptionsConfigurationError() {
+    this.errorMessage = `Custom sort options configuration is invalid.`;
```

Comment:
This needs to be a custom label instead added with its translation.
the rule is that any label displayed in the UI should be a custom label with translation,.